src/main/java/com/massivecraft/factions/P.java

```diff
@@ -117,8 +117,12 @@ public void onEnable() {
     }
 
     private boolean setupPermissions() {
-        RegisteredServiceProvider<Permission> rsp = getServer().getServicesManager().getRegistration(Permission.class);
-        perms = rsp.getProvider();
+        try {
+        	RegisteredServiceProvider<Permission> rsp = getServer().getServicesManager().getRegistration(Permission.class);
+        	perms = rsp.getProvider();
```

Comment:
If you're going to fix this, you may as well confirm the service provider isn't null. :)